bots/zephyr_mirror_backend.py

```diff
@@ -717,8 +751,11 @@ class and your mirroring bot does not have access to the relevant \
 %s
 
 %s""" % (support_heading, stderr, support_closing))
+    return
```

Comment:
That change looks good to me. It makes it more clear that the function returns nothing.